{
  "m_GameObject": {
    "m_FileID": 0,
    "m_PathID": 0
  },
  "m_Enabled": 1,
  "m_Script": {
    "m_FileID": 0,
    "m_PathID": -3078351595095444000
  },
  "m_Name": "335",
  "m_id": 335,
  "m_displayName": "CLAN QUEST - ORDER - TRISTEPIN - 05_Chapitre V",
  "m_availability": 2,
  "m_i18nNameId": {
    "a": -1530394942,
    "b": 48716228,
    "c": -755451414,
    "d": -802274573,
    "e": -1972025344
  },
  "m_i18nDescriptionId": {
    "a": -1530394942,
    "b": 48716228,
    "c": -755451414,
    "d": -802274573,
    "e": 345023637
  },
  "m_i18nDescriptionFinishedId": {
    "a": -1530394942,
    "b": 48716228,
    "c": -755451414,
    "d": -802274573,
    "e": -1231374874
  },
  "values": {
    "type": "AchievementDefinition",
    "id": 335,
    "subAchievements": [
      879
    ],
    "points": 10,
    "secret": false,
    "achievementType": 1,
    "rewards": [],
    "onSuccess": [],
    "precomputedData": {
      "type": "PrecomputedData",
      "dynamicValueReferences": [],
      "externalDynamicValueReferences": [],
      "hasEffects": false,
      "keywordReferences": [
        {
          "type": 1,
          "id": 7602
        },
        {
          "type": 1,
          "id": 251
        }
      ]
    }
  },
  "name": {
    "fr": "Compléter le chapitre V",
    "en": "",
    "es": "",
    "de": "",
    "pt": ""
  },
  "description": {
    "fr": "Avec son appeau, {Skin:251} semble avoir attiré un gros morceau. {Skin:7602} est prêt à descendre dans la faille pour régler le problème directement à la source.",
    "en": "",
    "es": "",
    "de": "",
    "pt": ""
  },
  "descriptionFinished": {
    "fr": "{Skin:7602}, {Skin:251} et {Skin:4446} ont croisé un monstre comme ils n’en avaient jamais vus. Il y a donc des choses inconnues qui rôdent dans les failles… Y a-t-il un lien avec l’entité apparue sur Astrub il y a peu\\_?",
    "en": "",
    "es": "",
    "de": "",
    "pt": ""
  }
}